src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1215,29 +1234,531 @@ public static VariantContext makeFromAlleles(final String name, final String con
         final int length = alleleStrings.get(0).length();
 
         boolean first = true;
-        for ( final String alleleString : alleleStrings ) {
+        for (final String alleleString : alleleStrings) {
             alleles.add(Allele.create(alleleString, first));
             first = false;
         }
-        return new VariantContextBuilder(name, contig, start, start+length-1, alleles).make();
+        return new VariantContextBuilder(name, contig, start, start + length - 1, alleles).make();
+    }
+
+    /**
+     * Split variant context into its biallelic components if there are more than 2 alleles
+     * <p>
+     * For VC has A/B/C alleles, returns A/B and A/C contexts.
+     * Alleles are right trimmed to satisfy VCF conventions
+     * <p>
+     * If vc is biallelic or non-variant it is just returned
+     * <p>
+     * Chromosome counts are updated (but they are by definition 0)
+     *
+     * @param vc                       a potentially multi-allelic variant context
+     * @param trimLeft                 if true, we will also left trim alleles, potentially moving the resulting vcs forward on the genome
+     * @param genotypeAssignmentMethod assignment strategy for the (subsetted) PLs
+     * @param keepOriginalChrCounts    keep the orignal chromosome counts before subsetting
+     * @return a list of bi-allelic (or monomorphic) variant context
+     */
+    public static List<VariantContext> splitVariantContextToBiallelics(final VariantContext vc, final boolean trimLeft, final GenotypeAssignmentMethod genotypeAssignmentMethod,
```

Comment:
I would keep this method but call into AlleleSubsettingUtils to perform most of the work.